domain-management/src/main/java/org/jboss/as/domain/management/security/LdapCacheService.java

```diff
@@ -356,6 +356,19 @@ private BaseSearchCache(final int evictionTime, final boolean cacheFailures, fin
                     }
                 }
             }
+            // Previous logic suppressed exceptions in finally statement,
```

Comment:
I don't believe this part is needed, the change to the finally block to eliminate the return statement should be sufficient.

Reply:
Thanks for your feedback, I'll reduce it.